pyproject.toml

```diff
@@ -4,12 +4,11 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length=100
-target-version=['py27']
+target-version=['py36']
```

Comment:
We can't do this yet until we officially drop py27. If you run black on files not modified by this PR, it'll strip `u` prefixes. Undo this change, and I'll take care of this during the pre-commit and black upgrade, along with a mass reformatting pass.

Reply:
Otherwise, this PR looks very thorough to me, good attention to detail here @armenzg.

Reply:
Thank you for the compliment and for catching this!